src/main/java/ch/kalunight/zoe/model/player_data/FullTier.java

```diff
@@ -84,6 +83,40 @@ public FullTier getHeigerDivision() {
       return new FullTier(tier, Rank.getRankWithValue(rank.getValue() + 100), 0);
     }
   }
+  
+  @Override
+  public int compareTo(FullTier otherFullTier) {

```

Comment:
Codacy found an issue: [The method 'compareTo(FullTier)' has an NPath complexity of 486, current threshold is 200](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5821508)